quickrun.el

```diff
@@ -1382,9 +1382,12 @@ by quickrun.el. But you can register your own command for some languages")
     (quickrun--add-remove-files dst)))
 
 (defun quickrun--kill-quickrun-buffer ()
-  "Kill quickrun buffer."
-  (when (get-buffer quickrun--buffer-name)
-    (kill-buffer quickrun--buffer-name)))
+  (let (quickrun--buffer (get-buffer quickrun--buffer-name))
```

Comment:
I think there is a syntax error over here? Can you check this? Thanks!

Reply:
Could you please merge this code in? 

I have similar issue when running `Quickrun` multiple times. This happens when piping `*quickrun*` buffer to a `popwin:special-display-config`

I can confirm this code fixed that issue.

Thanks.


@purcell

Reply:
How come this patch works? The variable `quickrun--buffer` will always return `nil` hence it will never run. 😕

Reply:
Not sure how/why, I don't know elisp. 

But, this fix does prevent opening multiple buffers on subsequent quickrun executions.